packages/server-extension/src/index.ts

```diff
@@ -373,6 +373,20 @@ const licensesRoutesPlugin: JupyterLiteServerPlugin<void> = {
   },
 };
 
+/**
+ * A plugin providing the routes for the lsp service.
+ * TODO: provide the service in a separate plugin?
```

Comment:
Hm... it already is a plugin... which is good, as it will make https://github.com/jupyterlite/jupyterlite-lsp/issues/1 _slightly_ less painful to get started back up.